src/Conference/Views/Shortcode/Schedule.php

```diff
@@ -0,0 +1,707 @@
+<?php
+/**
+ * Handles the schedule shortcode.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Conference\Views\Shortcode
+ */
+
+namespace TEC\Conference\Views\Shortcode;
+
+use TEC\Conference\Plugin;
+use WP_Query;
+
+/**
+ * Class Schedule
+ *
+ * @since   TBD
+ *
+ * @package TEC\Conference\Views\Shortcode
+ */
+class Schedule {
+
+	/**
+	 * Schedule Block and Shortcode Dynamic content Output.
+	 *
+	 * @since TBD
+	 *
+	 * @param array<string|mixed> $attr Array of attributes from shortcode.
+	 *
+	 * @return string The HTML output the shortcode.
+	 */
+	public function render_shortcode( $atts ) {
+		$defaults = [ 'date' => '' ];
```

Comment:
I removed the checks for Pro version as it will all be integrated and I updated the coding to PHP 7.4. 

There is a lot we could do here to clean up this coding further. 

But I would also like to keep moving on merging everything together. 

What about doing some essentials and then making a ticket to further improve this class... like changing all the html to load from a template, reducing the size of the render method... etc...